RealtimeTwitterToHBase/config.json

```diff
@@ -1,35 +1,69 @@
 {
+  "name": "RealtimeTwitterToHBase",
+  "artifact": {
+    "name": "cdap-data-streams",
+    "version": "3.5.1",
+    "scope": "SYSTEM",
+    "label": "Data Pipeline - Realtime"
+  },
+  "description": "",
   "config": {
+    "resources": {
+      "memoryMB": 512,
+      "virtualCores": 1
+    },
+    "driverResources": {
+      "memoryMB": 512,
+      "virtualCores": 1
+    },
+    "connections": [
+      {
+        "from": "Twitter",
+        "to": "tweetTable"
+      }
+    ],
+    "comments": [],
+    "postActions": [],
     "stages": [
       {
-        "name": "tableSink",
+        "name": "Twitter",
         "plugin": {
-          "name": "Table",
-          "type": "realtimesink",
+          "name": "Twitter",
+          "type": "streamingsource",
+          "label": "Twitter",
+          "artifact": {
+            "name": "spark-plugins",
+            "version": "1.4.1",
+            "scope": "SYSTEM"
+          },
           "properties": {
-            "name": "tweetTable",
-              "schema.row.field": "id",
-              "schema": "{\"type\":\"record\",\"name\":\"tweet\",\"fields\":[{\"name\":\"id\",\"type\":\"long\"},{\"name\":\"message\",\"type\":\"string\"},{\"name\":\"lang\",\"type\":[\"string\",\"null\"]},{\"name\":\"time\",\"type\":[\"long\",\"null\"]},{\"name\":\"favCount\",\"type\":\"int\"},{\"name\":\"rtCount\",\"type\":\"int\"},{\"name\":\"source\",\"type\":[\"string\",\"null\"]},{\"name\":\"geoLat\",\"type\":[\"double\",\"null\"]},{\"name\":\"geoLong\",\"type\":[\"double\",\"null\"]},{\"name\":\"isRetweet\",\"type\":\"boolean\"}]}"
+            "referenceName": "twitterSource",
+            "ConsumerKey": "GetConsumerKeyFromTwitter",
+            "ConsumerSecret": "GetConsumerSecretFromTwitter"
```

Comment:
missing a comma here. Guess that's why the ':' in the next line is highlighted in red?